### DISCUSSION TITLE
✨ Mosquito

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2189] [Requests]
### Description

Eclipse Mosquitto - An open source MQTT broker

### Official Website

https://mosquitto.org/

### Documentation link

https://mosquitto.org/documentation/

### Last application release & date

2.0.18 (18/09/2023)

### Application license

Eclipse Public License 2.0 / Eclipse Distribution License 1.0

### Source code repository link

https://github.com/eclipse/mosquitto

### Docker image link

https://hub.docker.com/_/eclipse-mosquitto

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @ceinem

**Reply by tommy-hartmann** [reply]
Done in https://github.com/runtipi/runtipi-appstore/pull/2549

**Reply by nicotsx** [reply]
Thanks @tommy-hartmann
